docs/get-started/v2/get-started/prebuilt/Screens-and-components.mdx

```diff
@@ -0,0 +1,105 @@
+- Select a role to start customising components on Preview and Room screens.
+
+<video loop="true" autoplay="autoplay" controls="controls" id="vid" muted>
+<source src="/docs/docs/v2/customiser.mp4" type="video/mp4" />
+</video>
+
+### Preview Screen
+
+<StepsContainer id="room-code-auth">
+
+#### Header
+Set up a custom header for each roles using `Title` and `Subtitle` fields.
+#### Join
+Choose how peers join the room
+	- **Join Now** - Peers from this role will see "Join Now" and would be able join after entering their name.
+	- **Join and Start Streaming** - Peers from this role will see "Go Live" and would be able to join and start streaming when they join the room. Ensure that this role has the permission to "Start/Stop HLS streaming" as `enabled` under Role permissions on its template configuration.
+
+</StepsContainer>
+
+### Room Screen
+Prebuilt's room screen provides components that enables interactivity an within a room. Select a role to start customising.
+
+#### Chat
+By default, Chat is enabled for all templates. Enable or disable chat functionality for a role. If enabled, you can further configure deeper functionalities on your Prebuilt UI.
+
+<StepsContainer id="room-code-auth">
+
+##### Initial State
+This defines initial of Chat component when a peer joins a room. 
+
+##### Enable Overlay View
+Enabling a chat overlay view The default state is set to false.
+
+##### Allow Pinning messages
+Enabling this allows the selected role to pin chat messages in your Prebuilt UI.  
+
+</StepsContainer>
+
+
+### Participant List
+Make participants list accessible for roles who need to see it. Scenarios where a role(for instance, viewers in a large room) don't need to see other participants, but a Host/Broadcasters would want to keep a tab on the participant list. By default, participant list is enabled by default for all roles for any template.
+
+
+### Video Tile Layout
+100ms Prebuilt UI allows tweaks on its default grid layout and supports multiple video layouts.
+
+<StepsContainer id="room-code-auth">
+#### Enable local tile inset
+Enabling this makes the local peer's *(for the selected role)* tile in a smaller inset window as default, alternatively if it's disabled, the local tile will be part of the grid view. Join with atleast two peers to preview this configuration in action.
+
+#### Prominent roles
+Defining one or more roles as prominent gives them higher tile view prominence for the selected role. For example:
+- For a 1:1 call, define Host as prominent tile for Guest and vice-versa. 
+- For a webinar, set Host as the prominent tile for all other roles. Every role would see Host as the primary tile in the room
+- For a mobile-first livestreaming view, set Broadcaster as prominent tile for every other role, including Broadcaster themselves.
+
+#### Can spotlight peer
+Allow this config for roles who can spotlight others or themselves in the room. Spotlighting a tile reflects for everyone in the room.
```

Comment:
explain how the UI experience of video tile is similar to the prominent role and also explain how this is slightly different

Reply:
done